services/movie_session.py

```diff
@@ -0,0 +1,47 @@
+from django.db.models import QuerySet
+from datetime import datetime
+
+from db.models import MovieSession

```

Comment:
Add a blank line between different import groups and order them correctly to follow PEP-8